scripts/train/train.py

```diff
@@ -25,6 +25,14 @@
                                            update_batch_size_info)
 
 
+def check_config_only_contains_special_params(cfg: DictConfig): 
+    """ Checks that the config is empty except for special parameters after we loaded everything in trainer. """
+    SPECIAL_UNPOPPED_PARAMETERS = ['data_local', 'data_remote', 'max_seq_len', 'global_seed', 'global_train_batch_size', 'n_gpus', 'device_train_grad_accum']
```

Comment:
I think we should figure out a way to make this not quite so brittly hardcoded. We can chat tomorrow